pkg/operator/ceph/object/bucket/s3-handlers.go

```diff
@@ -176,15 +176,14 @@ const (
 )
 
 // AllowedActions is a lenient default list of actions
-var AllowedActions = []action{
+var OBCAllowedActions = []action{
```

Comment:
Why do we need to rename this? Since it's part of the bucket package I think it's clear enough. Also if you really want to rename it you need to change the above comment accordingly.

Reply:
> Why do we need to rename this? Since it's part of the bucket package I think it's clear enough. Also if you really want to rename it you need to change the above comment accordingly.

Renaming is not really necessary, try to make it more clear since set of actions used as allow and deny in different circumstances